oxd-gen-client/src/main/java/io/swagger/client/ApiClient.java

```diff
@@ -712,11 +714,22 @@ public String escapeString(String str) {
             // Expecting string, return the raw response body.
             return (T) respBody;
         } else {
+            Gson gson = JSON.createGson().setPrettyPrinting().create();
+            ErrorResponse errorResponse = null;
+            try {
+                errorResponse = gson.fromJson(respBody, ErrorResponse.class);
+            } catch (Exception e) {
+                throw new ApiException(
+                        "Content type \"" + contentType + "\" is not supported for type: " + returnType,
+                        e,
+                        response.code(),
+                        response.headers().toMultimap());
+            }
```

Comment:
this is generated client, please don't write anything by hands. You can update it only from codegen.

Reply:
Changes reverted